src/app/components/Loop/index.tsx

```diff
@@ -0,0 +1,425 @@
+                  : 'Show advanced fields'}
+              </Button>
+            </Tooltip>
+          )}
+          <Form className="Loop" layout="vertical">
+            {url !== null && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Dropdown overlay={loopOutMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
+            {url !== null && loopType === 'Loop In' && (
+              <Form.Item>
+                <Dropdown overlay={loopInMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Input
+                  type="text"
+                  size="small"
+                  onChange={this.handleChangeDestination}
+                  placeholder="off-chain address"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item label="">
+                <Input
+                  size="small"
+                  onChange={this.handleChangeSwapRoutingFee}
+                  placeholder="swap routing fee"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && (
+              <Form.Item>
+                <Input
+                  width="50%"
+                  size="small"
+                  onChange={this.handleChangeSwapFee}
+                  placeholder="swap fee"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && (
+              <Form.Item>
+                <Input
+                  size="small"
+                  onChange={this.handleChangeMinerFee}
+                  placeholder="miner fee"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Input
+                  size="small"
+                  onChange={this.handleChangePrepayAmt}
+                  placeholder="prepay amt"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Input
+                  size="small"
+                  onChange={this.handleChangeConfTarget}
+                  placeholder="sweep confirmation target"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop In' && (
```

Comment:
You can replace all `advanced === true` checks with just `advanced &&...`

Reply:
replaced all advance === true to advanced &&